File: reto_14.py
#comparar un numero: GENERAR UNA LISTA DE LOS NUMEROS MAYORES QUE EL Y UNA LISTA DE LOS NUMEROS MENORES QUE EL
lista = input("Ingrese numeros separados por comas: ")
numero_a_comparar = input("Ingrese el numero a comparar: ")
lista_int = list(map(int, lista.split(",")))
numeros_mayores = []
numeros_menores = [] #menores o iguales
for item in lista_int:
    if item > int(numero_a_comparar):
        numeros_mayores.append(item)
    else:
        numeros_menores.append(item)

mayores = ", ".join(map(str,numeros_mayores))
menores = ", ".join(map(str,numeros_menores))
print(f"Los numeros mayores que {numero_a_comparar} son {mayores}. \nLos numeros menores o iguales que {numero_a_comparar} son {menores}.")

File: reto_14.js
//comparar un numero: GENERAR UNA LISTA DE LOS NUMEROS MAYORES QUE EL Y UNA LISTA DE LOS NUMEROS MENORES QUE EL
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,  
  output: process.stdout 
});
rl.question("Ingresa números separados por comas: ", (input) => {
    let lista = input.split(",");
    rl.question("Ingresa el numero a buscar: ", (input2) => {
        let numero_a_comparar = input2;
        let lista_int = lista.map(item => parseInt(item));
        let lista_mayores = [];
        let lista_menores = [];
        for (let item of lista_int){
            if(item > numero_a_comparar){
                lista_mayores.push(item);
            }else{
                lista_menores.push(item);
            }
        };
        let mayores = lista_mayores.join(", ");
        let menores = lista_menores.join(", ");
        console.log(`Los numeros mayores que ${numero_a_comparar} son ${mayores}.\nLos numeros menores o iguales que ${numero_a_comparar} son ${menores}.`)
    rl.close()
    });
});